app/controlplane/pkg/data/workflowcontract.go

```diff
@@ -397,10 +401,14 @@ func getWorkflowNames(ctx context.Context, schema *ent.WorkflowContract) ([]stri
 		}
 	}
 
-	names := make([]string, 0, len(workflows))
```

Comment:
this will break compatibility with current clients? Do you think it would be a problem? 

The reason I am asking is that in the API you mark `names` as deprecated instead of deleting it but in practice it will not return any data does it?

So we need to make a decision wether maintain both or remove it altogether, in general I'd say return both marking it as deprecated